display_menu.rb

```diff
@@ -0,0 +1,83 @@
+require_relative './exit_app'
+
+def display_menu(app)
+  puts "\nWelcome to our Catalog!"
+  puts "\nPlease choose a task by entering a number:"
+  puts " 1 - For listing items\n 2 - For adding items\n 3 - Exit App"
```

Comment:
Nice work displaying the options! For better organization, it would be better if the message 'Welcome to our Catalog!' doesn't appear after every loop.

Reply:
ok thanks